blobit-core/src/main/java/org/blobit/core/api/Configuration.java

```diff
@@ -185,6 +191,45 @@ public int getMaxEntrySize() {
                 MAX_ENTRY_SIZE_DEFAULT + ""));
     }
 
+    public Configuration setEmptyLedgerMinTtl(long value) {
+        properties.put(EMPTY_LEDGER_MIN_TTL, value + "");
+        return this;
+    }
+
+    /**
+     * Grace period for an empty ledger before beeing.
```

Comment:
Grace period for an empty ledger before beeing deleted?

Reply:
done, thanks